src/DocumentationBrowserViewExtension/DocumentationBrowserViewExtension.csproj

```diff
@@ -41,6 +41,7 @@
       <Project>{B5F435CB-0D8A-40B1-A4F7-5ECB3CE792A9}</Project>
       <Name>DynamoUtilities</Name>
     </ProjectReference>
+    <ProjectReference Include="..\LibraryViewExtensionMSWebBrowser\LibraryViewExtensionMSWebBrowser.csproj" />
```

Comment:
As you are using the interface from view extension manager, you don't need the doc browser to have a reference to the LibraryViewExtension, right?

Reply:
You are right, @reddyashish , removed as agreed!